docs/rules/no-duplicate-imports.md

```diff
@@ -1,6 +1,8 @@
 # Disallow duplicate imports (no-duplicate-imports)
 
-An ES6/ES2015 import can be spread over multiple lines, but this takes up unneeded whitespace. This rules validates that all imports from a single module exists in a single import statement.
+An ES6/ES2015 import can be spread over multiple lines, but this takes up unneeded whitespace.
```

Comment:
A question, not necessarily a request: Is the software quality goal of the rule clear enough without the first sentence?